cloud/openstack/os_compute_quotas.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/python
+
+# Copyright (c) 2015 Hewlett-Packard Development Company, L.P.
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+try:
+    import shade
+    HAS_SHADE = True
+except ImportError:
+    HAS_SHADE = False
+
+from distutils.version import StrictVersion
+
+
+DOCUMENTATION = '''
+---
+module: os_compute_quotas
+short_description: Update/Delete compute quotas in a OpenStack tenant.
+extends_documentation_fragment: openstack
+author: "Ghe Rivero (@GheRivero) and Andrea Mercanti (@mercantiandrea)"
+version_added: "2.3"
+description:
+   - Update or delete compute quotas in a OpenStack tenant.
```

Comment:
`tenant` is a deprecated term. Please use `project` instead. Also you've already got use of `project` in the docs, so be consistent.